Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPITabsCocoa.mm

```diff
@@ -635,7 +635,7 @@ static inline String toMIMEType(WebExtensionTab::ImageFormat format)
         auto injectedFrames = parameters.frameIDs ? WebCore::UserContentInjectedFrames::InjectInTopFrameOnly : WebCore::UserContentInjectedFrames::InjectInAllFrames;
 
         auto styleSheetPairs = getSourcePairsForParameters(parameters, *this);
-        injectStyleSheets(styleSheetPairs, webView, *m_contentScriptWorld, parameters.styleLevel, injectedFrames, *this);
+        injectStyleSheets(styleSheetPairs, webView, *RefPtr { m_contentScriptWorld }, parameters.styleLevel, injectedFrames, *this);
```

Comment:
`m_contentScriptWorld` is already a `RefPtr`.

Reply:
That's true--but per the C++ safe guidelines, we want to keep a smart pointer of an object on the stack when passing the object to a non-trivial function.

Reply:
same as earlier, I live `Ref { * m_contentScriptWorld }` though this is not a big deal.

Reply:
Fixed!